dev/source/docs/turnkey-companion-computer-solutions.rst

```diff
@@ -92,3 +92,35 @@ Full list can be `found here <https://www.dronotique.fr/docs/alfonce-remote-gate
 You can also read the full `documentation <https://www.dronotique.fr/docs/alfonce-remote-gateway/>`__ to discover full functionnalities
 
 By defaut it installed on a OrangePi Zero 2 + H5, but it could be integrated on a lot of supported nano computers or servers!
+
+XBStation - Platform For Internet Drone Base On Real Time 4G/5G Connectivity
+=========================================================
```

Comment:
The line size should be the same as line 96. Otherwise will display on Wiki build.